x-pack/plugins/apm/public/components/shared/search_bar.tsx

```diff
@@ -9,18 +9,29 @@ import React from 'react';
 import styled from 'styled-components';
 import { DatePicker } from './DatePicker';
 import { KueryBar } from './KueryBar';
+import { TimeComparison } from './time_comparison';
 
 const SearchBarFlexGroup = styled(EuiFlexGroup)`
   margin: ${({ theme }) =>
     `${theme.eui.euiSizeM} ${theme.eui.euiSizeM} -${theme.eui.gutterTypes.gutterMedium} ${theme.eui.euiSizeM}`};
 `;
 
-export function SearchBar(props: { prepend?: React.ReactNode | string }) {
+interface Props {
+  prepend?: React.ReactNode | string;
+  showTimeComparison?: boolean;
+}
+
+export function SearchBar({ prepend, showTimeComparison = false }: Props) {
   return (
     <SearchBarFlexGroup alignItems="flexStart" gutterSize="s">
       <EuiFlexItem grow={3}>
-        <KueryBar prepend={props.prepend} />
+        <KueryBar prepend={prepend} />
       </EuiFlexItem>
+      {showTimeComparison && (
+        <EuiFlexItem>
```

Comment:
```suggestion
        <EuiFlexItem style={{ minWidth: 300 }}>
```

This will help make sure the select field doesn't get squashed on smaller viewports.

_Before_
<img width="1419" alt="Screenshot 2021-01-20 at 15 00 08" src="https://user-images.githubusercontent.com/4104278/105185545-eb597b80-5b30-11eb-8af6-f14aebe7b6cf.png">

_After_
<img width="1418" alt="Screenshot 2021-01-20 at 15 05 56" src="https://user-images.githubusercontent.com/4104278/105185612-02986900-5b31-11eb-8c6b-414ba37a69da.png">